src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Search/SearchParameters/SearchParameterDefinitionManagerTests.cs

```diff
@@ -40,6 +43,8 @@ public class SearchParameterDefinitionManagerTests : IAsyncLifetime
         private readonly ISearchParameterSupportResolver _searchParameterSupportResolver;
         private readonly IFhirRequestContextAccessor _fhirRequestContextAccessor;
         private readonly IFhirRequestContext _fhirRequestContext = new DefaultFhirRequestContext();
+        private readonly SearchParameterStatusManager _searchParameterStatusManager;
```

Comment:
>SearchParameterStatusManager [](start = 25, length = 28)

why there is two of them? 
`private readonly SearchParameterStatusManager _manager;`
`private readonly SearchParameterStatusManager _searchParameterStatusManager;`